tests/handlers/test_koji.py

```diff
@@ -0,0 +1,65 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+from __future__ import unicode_literals
+
+import json
+from os import path
+from datetime import datetime
+
+from estuary.models.koji import KojiBuild, KojiTag
+import pytz
+
+from tests import message_dir
+from estuary_updater.handlers.koji import KojiHandler
+from estuary_updater import config
+
+
+def test_build_complete():
+    """Test the Koji handler when it recieves a new build complete message."""
+    with open(path.join(message_dir, 'koji', 'build_complete.json'), 'r') as f:
+        msg = json.load(f)
+    assert KojiHandler.can_handle(msg) is True
+    handler = KojiHandler(config)
+    handler.handle(msg)
+
+    build = KojiBuild.nodes.get_or_none(id_='736244')
+    assert build is not None
+    assert build.completion_time is None
+    assert build.creation_time == datetime(2018, 8, 3, 17, 49, 42, tzinfo=pytz.utc)
+    assert build.epoch is None
+    assert build.extra == ('{"source": {"original_url": "git://pkgs.devel.redhat.com/rpms/python-'
+                           'attrs?#3be3cb33e6432d8392ac3d9e6edffd990f618432"}}')
+    assert build.id_ == '736244'
+    assert build.name == 'python-attrs'
+    assert build.release == '8.el8+1325+72a36e76'
+    assert build.version == '17.4.0'
+    assert build.start_time == datetime(2018, 8, 3, 17, 49, 42, tzinfo=pytz.utc)
+    assert build.state == 0
+
+
+def test_build_tag():
+    """Test the Koji handler when it recieves a new build tag message."""
+    with open(path.join(message_dir, 'koji', 'build_tag.json'), 'r') as f:
+        msg = json.load(f)
+    assert KojiHandler.can_handle(msg) is True
```

Comment:
The build should exist before the tag so can you create a dummy build? You could reuse the pytest fixture we used in the Errata Tool and Freshmaker scraper.